pkg/controller/publishingstrategy/publishingstrategy_controller.go

```diff
@@ -136,7 +139,9 @@ func (r *ReconcilePublishingStrategy) Reconcile(request reconcile.Request) (reco
 	}
 
 	// ----------------------------TODO: remove these debug logs afterwards --------------------------------------
-	log.Info(fmt.Sprintf("initial ingresscontroller list: %+v", ingressControllerList.Items))
+	var serializedIngressControllerList bytes.Buffer
+	_ = serializer.Encode(ingressControllerList, &serializedIngressControllerList)
+	log.Info(fmt.Sprintf("initial ingresscontroller list: %s", serializedIngressControllerList.String()))
```

Comment:
Does this fail safe should the encoding fail for some reason?

Reply:
I don't expect that to happen since the object is fetched, but if it did then the `bytes.Buffer` would just be empty.